File: 58-length-of-last-word/length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int count = 0;
        String s1 = s.trim();
        int n = s1.length();
        for(int i = n - 1; i >= 0; i--){
            if(s1.charAt(i) == ' '){
                break;
            }else count++;
        }
        return count;
    }
}

File: 58-length-of-last-word/length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        n = len(s)
        i = n - 1
        lent = 0
        
        while i >= 0 and s[i] == ' ':
            i = i - 1
        while i >= 0 and s[i] != ' ':
            lent = lent + 1
            i = i - 1
        return lent